release notes/v0.52.0.md

```diff
@@ -0,0 +1,141 @@
+most likely make the experience of developing k6 extensions friendlier.
+
+### `lib.State` no longer has `Group` [#3750](https://github.com/grafana/k6/pull/3750)
+
+As the result of refactoring the implementation of `group` and `check` methods, in order to decouple them, and thus 
+enable other future improvements, the `lib.State` object no longer has a `Group` field.
+
+This change should not affect most users, except for a couple of extensions, for which the use of `Group` was
+already questionable:
+- `xk6-fasthttp`
+- `xk6-g0`
+
+### The `web-dashboard` built-in output becomes `dashboard` [xk6-dashboard#163](https://github.com/grafana/xk6-dashboard/pull/163)
+
+Since `v0.49.0`, k6 includes [xk6-dashboard](https://github.com/grafana/xk6-dashboard) as a built-in output extension.
+The name of the built-in module is the same as was the output name (`web-dashboard`) registered by the extension. 
+Therefore, if you try to use it as an extension (for example, a newer version), you will get the following error:
+
+```sh
+k6 run -o web-dashboard script.js
+
+ERRO[0000] invalid output extension web-dashboard, built-in output with the same type already exists
+```
+
+So, from now on, the registered name of the extension is again the one previously used: `dashboard`. 
+Thus, it is possible to use a version of the [xk6-dashboard](https://github.com/grafana/xk6-dashboard) that is newer than
+the one integrated in k6. In this case, the `dashboard` name must be used as the output name.
+
+### Other breaking changes
+
+- [browser#1318](https://github.com/grafana/xk6-browser/pull/1318) The `Mouse.up` and `Mouse.down` methods no longer take x and y coordinates. Instead, they dispatch events on the current mouse position.
+
+## New features
+
+### Experimental support for TypeScript and ES6+ using esbuild [#3738](https://github.com/grafana/k6/pull/3738)
+
+This release of k6 introduces experimental support for TypeScript and ES6+ using esbuild, thanks to a new
+[compatibility mode](https://grafana.com/docs/k6/latest/using-k6/javascript-compatibility-mode/) named `experimental_enhanced`.
+
+```sh
+k6 run --compatibility-mode=experimental_enhanced script.js
+```
+
+With this new compatibility mode, the test source code is transformed using esbuild instead of Babel, which also means
+that source files with the extension _".ts"_ are loaded by esbuild's TypeScript loader,  which results in partial 
+TypeScript support: it removes the type information, but does not provide type safety.
+
+### `k6/browser` has graduated from an experimental module [#3793](https://github.com/grafana/k6/pull/3793)
+
+The browser module is now available as `k6/browser` instead of `k6/experimental/browser`. The previous `k6/experimental/browser` module will be removed on September 23rd, 2024. Please see [the migration guide](https://grafana.com/docs/k6/latest/using-k6-browser/migrating-to-k6-v0-52/) for more information on how to update your scripts.
```

Comment:
```suggestion
The browser module is now available as `k6/browser` instead of `k6/experimental/browser`. The previous `k6/experimental/browser` module will be removed on September 23rd, 2024. Refer to [the migration guide](https://grafana.com/docs/k6/latest/using-k6-browser/migrating-to-k6-v0-52/) for more information on how to update your scripts.
```